ovos_tts_plugin_responsivevoice/__init__.py

```diff
@@ -21,15 +21,15 @@ class ResponsiveVoiceTTSPlugin(TTS):
     """Interface to ResponsiveVoice TTS."""
 
     def __init__(self, lang="en-us", config=None):
-        config = config or {"lang": lang,
+        self.config = config or {"lang": lang,
```

Comment:
`self.config` is assigned in the base class, dont assign to `self` here, but leave the `self.config` after the super call in pitch rate and volume